nucypher/blockchain/economics.py

```diff
@@ -66,9 +66,9 @@ class BaseEconomics:
 
     # Adjudicator
     _default_hash_algorithm = HASH_ALGORITHM_SHA256
-    _default_base_penalty = 100
-    _default_penalty_history_coefficient = 10
-    _default_percentage_penalty_coefficient = 8
+    _default_base_penalty = 2
+    _default_penalty_history_coefficient = 0
+    _default_percentage_penalty_coefficient = 100000  # 0.001%
```

Comment:
Does this mean slashing 0.001% if a misbehavior is detected?

Reply:
If so, I'd rather prefer something like 10 ** 18 here.

The reason is: an attacker who gained the access shouldn't be able to profit draining the pool (e.g. gas costs should outweigh his gains). With 0.001% it's not the case (if I understand correctly) if the stake is >~ $100k

Reply:
penalty will be allways 2 * 10^-18, I changed `_default_percentage_penalty_coefficient` for just in case but with those values it'll be unused